doc/changes.rst

```diff
@@ -10,7 +10,6 @@ Changelog
 
 .. raw:: latex
 
-   \hypersetup{bookmarksdepth=1}% pdf bookmarks
    \addtocontents{toc}{\protect\setcounter{tocdepth}{1}}%
```

Comment:
I drop the hypersetup, finally in bookmarks it is ok to access finest level for ChangeLog